RealtimeJMSToStream/README.rst

```diff
@@ -0,0 +1,85 @@
+Realtime Java Message Service (JMS) Source To Stream Sink Adapter Configuration
+===============================================================================
+
+The ETL Realtime Template can be used to create an Adapter that reads from a Realtime Source and pass it to a Sink to be processed further.
+In this example, we will read messages from Jave Message Service (JMS) source in realtime and use a Stream Sink to write the messages to CDAP Stream.
+
+The config.json contains a sample Adapter configuration that you can use to accomplish the above task. 
+
+Our sample Adapter uses the following components:
+
+- ETLRealtime Application Template, since we want to perform ETL in realtime
+- JMS source, using implementation of JMS from Apache ActiveMQ version 5.11.1
+- Stream sink to hold the incoming messages to be process by downstream program like CDAP Flows 
+
+You can create and start the Adapter by using the CDAP CLI (or you can use the UI for a more visual approach).
+
+The JMS source uses the JMS APIs to communicate with JMS server, so for The sample Adapter will use the Apache ActiveMQ version 5.11.1
+as the implementation of the JMS APIs. One important thing to remember is that you need to use the right version of JMS implementation
```

Comment:
you need to use the identical version of JMS for both the client and the target server to ensure that the message format is respected when the JMS source start polling messages.